libparsec/crates/platform_mountpoint/src/memfs.rs

```diff
@@ -202,7 +301,7 @@ impl MountpointInterface for MountpointManager {
             .to_str()
             .expect("Non utf8 name");
         parent.children.insert(
-            EntryName::try_from(file_name).expect("Should be a valid EntryName"),
+            EntryName::try_from(file_name).map_err(|_| MountpointError::NameTooLong)?,
```

Comment:
Pass the error to `NameTooLong`

PS: `EntryName` fail not only because the name is too long, we also look for invalid char ;) so rename the variant `NameTooLong` to `InvalidEntryName`

Reply:
Resolved